moveit/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.16.3)
```

Comment:
What's the oldest Ubuntu LTS release we're expected to support?

Reply:
There is a REP about [ROS 2 Releases and Target Platforms ](https://www.ros.org/reps/rep-2000.html). I've chosen the minimal version based on that

Reply:
> A single ROS 2 distribution will only have full Tier 1 support for a single Ubuntu LTS. The rationale is that fully supporting two LTS versions - which means 2-year-different versions of upstream dependencies - is a tremendous overhead and sometimes even impossible. 

If we're only supporting Ubuntu 22 with this code then you can raise that CMake requirement to 3.22.

Reply:
This is actually a good point, I'll bump the version to 3.22. @henningkayser @vatanaksoytezer @JafarAbdi any objections?

Reply:
You can safely bump this to 3.22, we don't support Focal on `main` or `humble`.